test_data/curried_fn.good.nix

```diff
@@ -1,6 +1,8 @@
 {
   mkValueStringDefault = {}: v: with builtins;
-    let err = t: v: abort
-      "oups";
-    in 92;
```

Comment:
@zimbatm not sure what rule do we want here. The current implementation enforces line breaks if let itself is multiline, but we can allow `in 92` to be on the single line.

Reply:
I think it's fine like that